modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/Podcast.kt

```diff
@@ -204,15 +205,17 @@ data class Podcast(
         val sevenDaysInMs = 7 * DateUtils.DAY_IN_MILLIS
         val now = System.currentTimeMillis()
         val sevenDaysAgo = now - sevenDaysInMs
-        val sevenDaysFuture = now + sevenDaysInMs
+
+        val resources = context.resources
         return when {
             expectedTime < sevenDaysAgo -> null
             DateUtils.isToday(expectedTime) -> resources.getString(LR.string.podcast_next_episode_today)
             DateUtils.isToday(expectedTime - DateUtils.DAY_IN_MILLIS) -> resources.getString(LR.string.podcast_next_episode_tomorrow)
             expectedTime in sevenDaysAgo..now -> resources.getString(LR.string.podcast_next_episode_any_day_now)
```

Comment:
Since we show "any day now" for a date between now and 7 days ago, 
`expectedTime in sevenDaysAgo..now -> resources.getString(LR.string.podcast_next_episode_any_day_now)`

we might want to reduce time period from
`calendar.after(sevenDaysAgo) && calendar.before(sevenDaysFromNow) -> DateUtils.FORMAT_SHOW_WEEKDAY`
to 
`calendar.after(now) && calendar.before(sevenDaysFromNow) -> DateUtils.FORMAT_SHOW_WEEKDAY`
inside [RelativeDateFormatter](https://github.com/Automattic/pocket-casts-android/pull/477/commits/4328b37a725852c18d7dca5af12986248b150a95#diff-1715dbdb50e2293050b7f90f7e612032209eab4e476fd622aa9677bb9f37df6cR50). 

Also, we might want to match [iOS date formatting](https://github.com/Automattic/pocket-casts-ios/blob/trunk/podcasts/Podcast+Formatting.swift#L55-L56) and instead of adding seven days to now, we can just add six days to be consistent in both platforms.